### PR TITLE
[TAS-1067] 💄 Add recommendation section to claim page

### PR DESCRIPTION
**Pull request by AuroraHuang22**
based on #1529

網頁擷取_29-1-2024_234843_localhost
網頁擷取_29-1-2024_234928_localhost

### CODE REVIEW

File: src/pages/nft/claim/index.vue [hunk 1]
```diff
@@ -784,6 +811,51 @@ export default {
...
+        classId,
+        1
+      );
+    },
+    handleRecommendationSlideNext() {
+      logTrackerEvent(
+        this,
+        'NFT',
+        'nft_claim_recommendation_clicked_next',
+        this.classId,
```
Comment (williamchong) [inline src/pages/nft/claim/index.vue:837]:
```suggestion
        this.primaryKey,
```

File: src/pages/nft/claim/index.vue [hunk 2]
```diff
@@ -784,6 +811,51 @@ export default {
...
+        1
+      );
+    },
+    handleRecommendationSlidePrev() {
+      logTrackerEvent(
+        this,
+        'NFT',
+        'nft_claim_recommendation_clicked_prev',
+        this.classId,
```
Comment (williamchong) [inline src/pages/nft/claim/index.vue:846]:
```suggestion
        this.primaryKey,
```

File: src/pages/nft/claim/index.vue [hunk 3]
```diff
@@ -784,6 +811,51 @@ export default {
...
+        1
+      );
+    },
+    handleRecommendationSliderMove() {
+      logTrackerEvent(
+        this,
+        'NFT',
+        'nft_claim_recommendation_moved_slider',
+        this.classId,
```
Comment (williamchong) [inline src/pages/nft/claim/index.vue:855]:
```suggestion
        this.primaryKey,
```

File: src/pages/nft/claim/index.vue [hunk 4]
```diff
@@ -503,6 +529,7 @@ export default {
     if (this.isFreePurchase) {
       this.claimingAddress = this.claimingAddressInput;
     }
+    await this.fetchRecommendInfo();
```
Comment (williamchong) [inline src/pages/nft/claim/index.vue:532]: Should we refetch after use confirmed `claimingAddress` ?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/pages/nft/claim/index.vue (hunk 1 above)
Comment (AuroraHuang22) [reply src/pages/nft/claim/index.vue]: [📈 Update logger property](https://github.com/likecoin/liker-land/pull/1532/commits/646370555e000dc7c94b23a7a9d753a862a47eb8)

File: src/pages/nft/claim/index.vue (hunk 4 above)
Comment (AuroraHuang22) [reply src/pages/nft/claim/index.vue]: [🚸 Refetch recommendInfo after changing claimingAddress](https://github.com/likecoin/liker-land/pull/1532/commits/674ea554038d2941a1190a40144f7abbfc43ef7e)

**Review by williamchong — APPROVED** [review]

**Pull request merged**